unit-tests-java8/src/test/java/org/eclipse/collections/test/bag/mutable/sorted/MutableBagIterableTestCase.java

```diff
@@ -45,9 +46,9 @@ default void MutableBagIterable_removeOccurrences_throws()
     default void MutableBagIterable_addOccurrences()
     {
         MutableBagIterable<Integer> mutableBag = this.newWith(1, 2, 2, 3, 3, 3);
-        mutableBag.addOccurrences(4, 4);
+        Assert.assertEquals(4, mutableBag.addOccurrences(4, 4));
```

Comment:
The test uses a combination of static imports and plain imports for assertions. In this case, I'd recommend static imports for consistency. No need for new imports.